lib/data_update_scripts/20200217131245_re_index_existing_articles_with_approved.rb

```diff
@@ -1,7 +1,7 @@
 module DataUpdateScripts
   class ReIndexExistingArticlesWithApproved
     def run
-      Article.published.find_each { |article| Article.trigger_index(article, false) }
+      # Article.published.find_each { |article| Article.trigger_index(article, false) }
```

Comment:
as `trigger_index` is deleted, maybe we can remove the body altogether (as uncommenting this line would break anyway)

Reply:
replaced with a brief comment!